cli/definitions/cli.ts

```diff
@@ -113,7 +113,6 @@ export interface ILoginCommand extends ICommand {
 }
 
 export interface ILogoutCommand extends ICommand {
-    isLocal: boolean;
 }
```

Comment:
ICommand has the "type" field.  That's all you really need to call the logout(...) function.  The ILogoutCommand interface is otherwise useless.

With your change the cast on line 457 is no longer necessary, and the `command` parameter on line 559 isn't even used.